pom.xml

```diff
@@ -707,6 +707,11 @@
       </dependencyManagement>
 
       <dependencies>
+        <dependency>
```

Comment:
Check if this is required to be submitted in the code.

Reply:
Yes we need these changes for sl4j errors